### ISSUE TITLE
Spike: Investigate reducing noise in logs

### ISSUE THREAD

**Issue by dimberman** [astro-projects/astro#155]
Right now logs in astro tasks are VERY noisy due to the sheer number of logs in certain connectors (e.g. snowflake, BQ, etc.). Do we actually need to surface all of these logs?

**Comment by dimberman** [comment]
@kaxil what are your thoughts here? On the one hand it feels risky to suppress connector logs (e.g. if the SnowflakeHook thinks these are INFO level then maybe they are?)

**Comment by kaxil** [comment]
Yeah I agree, we should not suppress them and users can change the Airflow log level too
